scripts/alpha_encimg.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+#
+# encrypt factory images for D-Link DIR-842, DIR-859
+#
+# Copyright (c) 2020 Sebastian Schaper <openwrt@sebastianschaper.net>
+#
+# License GPLv2+: GNU GPL version 2 or later <http://gnu.org/licenses/gpl-2.0.html>
+#
+
+from Crypto.Cipher import AES
```

Comment:
This import would result in error on our build slaves

```
$ docker run --rm -it openwrtorg/buildslave:latest python3 -c "from Crypto.Cipher import AES"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named 'Crypto'
```

Reply:
> This import would result in error on our build slaves

That's good to know, thanks. While the code was tested against both the legacy PyCrypto as well as PyCryptodome, I was wondering whether the buildbots actually use the same "full" python3 installation as if I were to `apt install python3` e.g. on ubuntu.

However `libssl-dev` is listed as a dependency as well, so I presume it should work out of the box when including the AES functions from openssl within C code?

Reply:
Well, you can run the container yourself `docker run --rm -it openwrtorg/buildslave:latest /bin/bash` and check what is there or missing, the Dockerfile is here https://git.openwrt.org/?p=buildbot.git;a=blob;f=docker/buildslave/Dockerfile;h=96919bdbb3dada21eb0a0e8ad564a0f7fb135f50;hb=HEAD